CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 * None
 
 ### Enhancements
+* Added new error class `CompensatingWriteError`, that that indicates that one or more object changes have been reverted by the server. 
+This can happen when the client creates/updates objects that do not match any subscription, or performs writes on an object it didn't have permission to ([#5599](https://github.com/realm/realm-js/pull/5599)).
```

Comment:
```suggestion
This can happen when the client creates/updates objects that do not match any subscription, or performs writes on an object it didn't have permission to access. ([#5599](https://github.com/realm/realm-js/pull/5599))
```